dns/bind/src/opnsense/scripts/OPNsense/Bind/dnsbl.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
```

Comment:
this probably needs a linked cron job to set this up and run it regularly

Reply:
btw, this file should have your copyright

Reply:
looks good, can we maybe remove "all" and implement multiselect on top after this PR? it's simple to do with a working base setup